src/index.js

```diff
@@ -4,6 +4,8 @@ import {getContainer} from './utils'
 const getDefaultCommandOptions = () => {
   return {
     timeout: Cypress.config().defaultCommandTimeout,
+    fallbackToPreviousFunctionality: true,
```

Comment:
Thoughts? I figured this could be default behavior to avoid any backwards-breaking changes. This feature isn't currently available through Typescript definitions and maybe shouldn't be. This config can disable the backwards compatibility mode required for #110

Reply:
I like it. Trying to think of a better name for it though. How about: `enableChainContinuation`?

Another thing to consider is maybe people would like to configure this globally rather than on a per-command basis. Should we expose a mechanism for changing the default config? Similar to how DOM Testing Library works?

Reply:
> I like it. Trying to think of a better name for it though. How about: `enableChainContinuation`?

Hmm. I agree `fallbackToPreviousFunctionality` isn't a good name because it is unclear what it does. `enableChainContinuation` isn't very descriptive and I interpret it as not what is actually happening.

By default all queries are trying to use previous subject (like Cypress DOM commands do except `cy.get`) and if an error is encountered, like `selector.querySelectorAll is not a function`, it will try again without using the previous subject. Setting this property to `false` will disable that fallback. What about `retryWithoutPreviousSubject` or `fallbackRetryWithoutPreviousSubject`?

> Another thing to consider is maybe people would like to configure this globally rather than on a per-command basis. Should we expose a mechanism for changing the default config? Similar to how DOM Testing Library works?

I was thinking the same thing. Wrapping the `config` from DOM Testing Library and extracting this preference. I'll work to add that. 👍

Reply:
I like `retryWithoutPreviousSubject` more. Thanks!

Reply:
I realized that `configure` is untested and I'm not sure if other people have used it, but if you `import` the `configure` function from `@testing-library/cypress`, you will actually get a different reference that what is used inside query commands. This happens because Cypress is actually running outside your application.

We would have to expose a configure command instead:

```ts
cy.configureCypressTestingLibrary(config)
```

This command would then have access to the same reference... I'm guessing this is either an issue or people haven't configured `@testing-library/dom` yet

Reply:
That's fine with me 👍

I doubt people are configuring anything today.

Reply:
I'm currently validating if my assumptions are correct. What I said is definitely true of importing code from the application into the test, but may not be true in this case. I'll add tests to verify.

Reply:
I have verified...
![Screen Shot 2020-01-31 at 4 19 51 PM](https://user-images.githubusercontent.com/338257/73581153-89969200-4445-11ea-912d-8a86d79c01b7.png)

That's from a single `console.log` in the `index.js` file. Once is called from the import from `support.js` (where the commands are registered) and the other imported from the test file. They are not the same references, I'll have to add a command to allow for configuring